samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -44,63 +69,14 @@ private void Update()
         {
             mapComponent.APIKey = APIKey;
         }
+
+        SceneLoadedCount = SceneManager.sceneCount;
     }
 
-    private void Start()
+    // Read string from the input field for the API key
+    public void ReadStringInput(string s)
```

Comment:
Avoid abbreviated or one letter argument names, giving the arg name something descriptive makes the code readable  

```suggestion
    public void ReadStringInput(string apiKey)
```